### DISCUSSION TITLE
Debug mode cannot be deactivated.

### DISCUSSION THREAD

**Discussion by hatty24** [TandoorRecipes/recipes#3221] [Q&A]
Hello everyone,

I have installed Tandoor with Docker Compose as described in the instructions. Everything is running very well.
But when I go into the system settings, it always shows that debug mode is running.

<img width="1662" alt="Bildschirmfoto 2024-07-18 um 11 48 38" src="https://github.com/user-attachments/assets/8d4ee71b-2671-46f7-a9c8-7dc5c324af54">

I have not activated it myself. There was also no entry in the default file for Docker Compose.

Although I added the line DEBUG=0 to the .env file as described in the message and restarted the container, I still get the message.

Does anyone have any idea what else I can do to switch off the debug mode?

**Comment by smilerz** [comment]
Changes in the env require rebuilding the container, not just restart.

**Comment by hatty24** [comment]
Okay.
Rebuilding the container means backing up the recipes and ingredients beforehand, setting up the container again, and then restoring everything, right?
That seems like a lot of effort just to get rid of the message.
What is the disadvantage of simply leaving the system running in debug mode?

**Reply by smilerz** [reply]
Containers are stateless - rebuilding just means building the container with the new settings.

`docker-compose up --force-recreate`

That being said, there is nothing wrong with leaving it in debug mode - on low resource systems you might see performance changes, but I doubt it.

**Comment by hatty24** [comment]
Great!
Thanks for your support.
